apps/examples/sensorbd_demo/examples/spi_lis3lv02qd.c

```diff
@@ -117,18 +117,18 @@ void lis3lv02qd_main(int argc, char *argv[])
 		return;
 	}
 
- 	uint16_t x_low, x_high, y_low, y_high, z_low, z_high;
+	uint16_t x_low, x_high, y_low, y_high, z_low, z_high;
 	uint32_t x = 0, y = 0, z = 0;
 
 	int i;
 	for (i = 0; i < 30; i++) {
 		spi_write(port, LIS3LV02DQ_CONF_REG_1, freq, bits, conf, 0x87);
 
-		x_low  = spi_read(port, LIS3LV02DQ_OUTX_L, freq, bits, conf);
+		x_low = spi_read(port, LIS3LV02DQ_OUTX_L, freq, bits, conf);
```

Comment:
The original author put spaces to align the equal signs in lines. Does TizenRT coding standard explicitly prohibit extra spaces? Really?